.github/workflows/wiki.yml

```diff
@@ -1,5 +1,8 @@
 name: Deploy to Wiki
 
+permissions:
```

Comment:
@BenHenning Generally (at least on Web), we are moving towards a system where all yml files specify "permissions: {}" at the top of the workflow file, and individual jobs are assigned higher permission levels. This allows us to isolate elevated permissions to specific jobs rather than an entire workflow.

See https://github.com/oppia/oppia/blob/develop/.github/workflows/full_stack_tests.yml#L298 for an example (I know it's permissions: read-all at the top but we do want to change that to {} as well).

Should we adopt a similar approach on the Android team?

Reply:
I'm completely fine with making the permission at a job-level.

I don't see value in marking the workflow as permissions {}, though. Simply omitting it will set all of the jobs to the organization-level default which we've deemed is reasonable. I'm generally not keen on the human element of having to remember to include the "reset permissions" line (though I am a fan of restricting access, so I'll go ahead and make the corresponding change here).

Reply:
Marking as resolved since I think you're fine with this per your latest comment.